USAGE.md

```diff
@@ -0,0 +1,107 @@
+The skill is supported for English (U.S.), English (U.K.), and German languages. 
+
+# General configuration Instructions
+
+## Requirements
+
+* myopenHAB.org cloud service running on your instance of openHAB2
+* Amazon accout
+* Echo, Echo Dot, or another way to use Alexa (e.g. [Reverb](https://reverb.ai/))
+<!--- did I miss something? --->
+
+## 1. Skill Configuration
+
+1. Visit the [Alexa-Website](https://alexa.amazon.com/) or use the belonging app on your mobile.
+2. Navigate to "Smart Home" -> "Configure Smart Home" and search for openHAB.
+3. Follow the Instructions there to autohorise Alexa using your myopenhab.org-account.
+
+If you did not "tag" any items until now, first follow the steps below before you search for devices.
+(You can try, but you will not find any ;) )
+
+## 2. Item Configuration
+
+You have to tag your items to make them useable by Alexa.
+See [Tagging](http://docs.openhab.org/configuration/items.html#tagging)
+
+Since tagging (actualy) is a new feature it differs by the way you manage your items.
+
+* **Items via .items - file**
+
+  Some examples of tagged items are:
+  
+  ```java
+  Switch KitchenLights "Kitchen Lights" <light> (gKitchen) [ "Lighting" ]
+  Dimmer BedroomLights "Bedroom Lights" <light> (gBedroom) [ "Lighting" ]
+  Number BedroomTemperature "Bedroom Temperature" (gBedroom) [ "CurrentTemperature" ]
+  
+  Group gDownstairsThermostat "Downstairs Thermostat" (gFF) [ "Thermostat" ]
+  Number DownstairsThermostatCurrentTemp "Downstairs Thermostat Current Temperature" (gDownstairsThermostat) [ "CurrentTemperature" ]
+  Number DownstairsThermostatTargetTemperature "Downstairs Thermostat Target Temperature" (gDownstairsThermostat) [ "TargetTemperature" ]
+  String DownstairsThermostatHeatingCoolingMode "Downstairs Thermostat Heating/Cooling Mode" (gDownstairsThermostat) [ "homekit:HeatingCooling
+  ```
+  
+  Please make sure to place your tag infront of the channel-informations. e.g.
+  
+   ```java
+   Switch KitchenLights "Kitchen Lights" <light> (gKitchen) [ "Lighting" ] {channel="..."}
+   ```
+   
+* **Items via PaperUI**
+ 
+   There (actually) is no easy way to tag yout Items via PaperUI.
+ 
+   Some users managed to tag items via REST API.
+   Have a loot at [this post](https://community.openhab.org/t/apply-tags-to-items-added-linked-in-paper-ui/19443/11?u=mboremski).
+   <!--- Are there more relevant ways to configure items? --->
+   <!--- Should we add a chapter for availabletags? --->
+
+## 3. Final Configuration
+
+After tagging your items you can go back to the Alexa-configurations and search for devices.
+
+## Example Voice Commands
+
+Here are some example voice commands:
+
+ * Alexa turn on Office Lights
+ * Alexa turn off Pool Waterfall
+ * Alexa turn on House Fan
+ * Alexa turn on Home Theater Scene
+ * Alexa dim Kitchen Lights to 30 percent
+ * Alexa set house temperature to 70 degrees
```

Comment:
Put in quotes?

Also I believe it should read "Alexa, turn on the office light".

Besides I'd prefer examples linked to actual openHAB items so the reader immediately understands the possibilities. Maybe you can adapt these examples to use the items shown above?

Reply:
Shame on me. I also copied that..
I can't try the english expressions because I use the german Alexa...

Ill do the rework belonging to the examples above.